static_report/rollup.config.mjs

```diff
@@ -0,0 +1,58 @@
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import typescript from '@rollup/plugin-typescript';
+import alias from '@rollup/plugin-alias';
+import dts from 'rollup-plugin-dts';
+import { terser } from 'rollup-plugin-terser';
+import peerDepsExternal from 'rollup-plugin-peer-deps-external';
+import nodePolyfills from 'rollup-plugin-node-polyfills';
+import { readFileSync } from 'fs';
+import path from 'path';
+
+const packageJson = JSON.parse(
+  readFileSync(new URL('./package.json', import.meta.url)),
```

Comment:
I remember you can import `package.json` directly:

```js
import pkg from './package.json';
```

So you don't need to use `readFileSync`.

Reply:
I thought so too, but i encountered `type: module` errors while doing that with a `*.mjs` file: 
Rollup docs: 
![image](https://user-images.githubusercontent.com/11186480/191628223-71a9351b-f430-41fc-8a37-1b9f5b84d449.png)